cdc/log.cc

```diff
@@ -978,7 +982,12 @@ class log_mutation_builder {
     // Given a reference to such a column from the base schema, this function sets the corresponding column
     // in the log to the given set of keys for the given row.
     void set_deleted_elements(const clustering_key& log_ck, const column_definition& base_cdef, const managed_bytes& deleted_elements) {
```

Comment:
There's also the `set_deleted` function. Fortunately, the overload used there does check whether the column exists or not and throws a runtime_error, so we do not risk a crash here. However, I think it would be good to throw an error with the similar message as the one for `set_value` and `set_deleted_elements`.